modules/odc-deploying-java-applications.adoc

```diff
@@ -2,40 +2,42 @@
 [id="odc-deploying-java-applications_{context}"]
 = Upload JAR files for easy deployment of Java applications
 
```

Comment:
The module is missing some file metadata. It misses the information about the assembly in which the module is included:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata